phoenix-core/src/main/java/org/apache/phoenix/hbase/index/metrics/GlobalIndexCheckerSource.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.hbase.index.metrics;
+
+import org.apache.hadoop.hbase.metrics.BaseSource;
+
+/**
+ * Interface for metrics from GlobalIndexChecker
+ */
+public interface GlobalIndexCheckerSource extends BaseSource {
+    // Metrics2 and JMX constants
+    String METRICS_NAME = "GlobalIndexChecker";
+    String METRICS_CONTEXT = "phoenix";
```

Comment:
Would it be better to be more specific something like phoenixSecondaryIndex?

Reply:
Not sure. I just want to avoid repeating 'phoenix' and 'index' everywhere and make the fully qualified metric name extremely long. 'secondary' is sort of implied everywhere when we say index, right?

Reply:
I'll confirm once I get a test run and see the metrics from JMX port. We can decide accordingly.